src/crashreporter/main.cpp

```diff
@@ -24,10 +24,6 @@
 
 int main(int argc, char *argv[])
 {
-    QCoreApplication::setAttribute(Qt::AA_UseHighDpiPixmaps, true);
```

Comment:
Is this related to the change? Does this really fix the crash? If so, I'd like to know how (to avoid this in the future).

Reply:
The flags are deprecated.
I will create a follow up pr which will apply our compiler settings to the crash reporter.